driver/modern_bpf/helpers/extract/extract_from_kernel.h

```diff
@@ -130,6 +130,43 @@ static __always_inline void extract__ino_from_fd(s32 fd, u64 *ino)
 	BPF_CORE_READ_INTO(ino, f, f_inode, i_ino);
 }
 
+/**
+ * @brief Return the `f_inode` of task exe_file.
+ *
+ * @param mm pointer to task mm struct.
+ * @return `f_inode` of task exe_file.
+ */
+static __always_inline struct inode *extract__exe_inode_from_task(struct mm_struct *mm)
+{
+	struct inode *inode = NULL;
+	if (mm)
+	{
+		struct file *exe_file;
+		BPF_CORE_READ_INTO(&exe_file, mm, exe_file);
+		if (exe_file)
+		{
+			BPF_CORE_READ_INTO(&inode, exe_file, f_inode);
+		}
+	}
+	return inode;
```

Comment:
this could become a one-liner

```suggestion
	return BPF_CORE_READ(mm, exe_file, f_inode);
```